src/octokit.ts

```diff
@@ -44,15 +52,15 @@ export interface IRepositoryInfo {
 const extractSha = (eventName: string, event: any): E.Either<Error, string> => {
```

Comment:
@XVincentX Why are you using `event: any` and not `event: Event` here? 
Is `event.pull_request` ever valid?

Reply:
Because apparently the definitions are wrong; I'm gonna try to take another look into but we'd need a decoder for this instead of relying on the typings.